run-local.sh

```
@@ -0,0 +1,76 @@
+# Collect static files
+echo "Collecting static files..."
+python manage.py collectstatic --noinput
+
+# Start Django server in background
+echo "Starting Django server..."
+python manage.py runserver 8000 &
+API_PID=$!
+
+# Setup UI
+echo "Setting up UI..."
+cd ../ui
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)